diff --git a/node_modules/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.mm b/node_modules/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.mm
index b85095d..9328efd 100644
--- a/node_modules/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.mm
+++ b/node_modules/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.mm
@@ -7,7 +7,7 @@
 #import <ExpoModulesCore/RCTAppDelegateUmbrella.h>
 
 #import <React/RCTComponentViewFactory.h> // Allows non-umbrella since it's coming from React-RCTFabric
-#import <ReactCommon/RCTHost.h> // Allows non-umbrella because the header is not inside a clang module
+#import <react/runtime/platform/ios/ReactCommon/RCTHost.h> // Allows non-umbrella because the header is not inside a clang module
 
 // TODO remove the if when 76 is not supported, or rather remove the EXAppDelegateWrapper because it's deprecated
 #if __has_include(<ReactAppDependencyProvider/RCTAppDependencyProvider.h>)
diff --git a/node_modules/expo-modules-core/ios/Core/ExpoBridgeModule.mm b/node_modules/expo-modules-core/ios/Core/ExpoBridgeModule.mm
index 2ed1c00..24131ad 100644
--- a/node_modules/expo-modules-core/ios/Core/ExpoBridgeModule.mm
+++ b/node_modules/expo-modules-core/ios/Core/ExpoBridgeModule.mm
@@ -1,12 +1,12 @@
 // Copyright 2024-present 650 Industries. All rights reserved.
 
-#import <ReactCommon/RCTTurboModule.h>
+#import "../../../react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModule.h"
 #import <ExpoModulesCore/ExpoBridgeModule.h>
 #import <ExpoModulesCore/Swift.h>
 
 // The runtime executor is included as of React Native 0.74 in bridgeless mode.
-#if __has_include(<ReactCommon/RCTRuntimeExecutor.h>)
-#import <ReactCommon/RCTRuntimeExecutor.h>
+#if __has_include(<react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>)
+#import <react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>
 #endif // React Native >=0.74
 
 @implementation ExpoBridgeModule
@@ -45,12 +45,12 @@ - (void)setBridge:(RCTBridge *)bridge
   _bridge = bridge;
   _appContext.reactBridge = bridge;
 
-#if !__has_include(<ReactCommon/RCTRuntimeExecutor.h>)
+#if !__has_include(<react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>)
   _appContext._runtime = [EXJavaScriptRuntimeManager runtimeFromBridge:bridge];
 #endif // React Native <0.74
 }
 
-#if __has_include(<ReactCommon/RCTRuntimeExecutor.h>)
+#if __has_include(<react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>)
 - (void)setRuntimeExecutor:(RCTRuntimeExecutor *)runtimeExecutor
 {
   _appContext._runtime = [EXJavaScriptRuntimeManager runtimeFromBridge:_bridge withExecutor:runtimeExecutor];
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.h b/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.h
index b26eb1c..dc840a9 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.h
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.h
@@ -6,7 +6,7 @@
 @class EXAppContext;
 @class EXRuntime;
 
-#if __has_include(<ReactCommon/RCTRuntimeExecutor.h>)
+#if __has_include(<react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>)
 @class RCTRuntimeExecutor;
 #endif // React Native >=0.74
 
@@ -23,7 +23,7 @@ extern NSString * _Nonnull const EXGlobalCoreObjectPropertyName;
  */
 + (nullable EXRuntime *)runtimeFromBridge:(nonnull RCTBridge *)bridge NS_SWIFT_NAME(runtime(fromBridge:));
 
-#if __has_include(<ReactCommon/RCTRuntimeExecutor.h>)
+#if __has_include(<react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>)
 + (nullable EXRuntime *)runtimeFromBridge:(nonnull RCTBridge *)bridge withExecutor:(nonnull RCTRuntimeExecutor *)executor;
 #endif // React Native >=0.74
 
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.mm b/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.mm
index d0f8396..0556021 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.mm
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIInstaller.mm
@@ -39,7 +39,7 @@ + (nullable EXRuntime *)runtimeFromBridge:(nonnull RCTBridge *)bridge
   return jsiRuntime ? [[EXRuntime alloc] initWithRuntime:jsiRuntime callInvoker:bridge.jsCallInvoker] : nil;
 }
 
-#if __has_include(<ReactCommon/RCTRuntimeExecutor.h>)
+#if __has_include(<react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>)
 + (nullable EXRuntime *)runtimeFromBridge:(nonnull RCTBridge *)bridge withExecutor:(nonnull RCTRuntimeExecutor *)executor
 {
   jsi::Runtime *jsiRuntime = reinterpret_cast<jsi::Runtime *>(bridge.runtime);
