diff --git a/node_modules/react-native-reanimated/apple/reanimated/apple/LayoutReanimation/REASnapshot.m b/node_modules/react-native-reanimated/apple/reanimated/apple/LayoutReanimation/REASnapshot.m
index 893aeba..21c5313 100644
--- a/node_modules/react-native-reanimated/apple/reanimated/apple/LayoutReanimation/REASnapshot.m
+++ b/node_modules/react-native-reanimated/apple/reanimated/apple/LayoutReanimation/REASnapshot.m
@@ -2,8 +2,8 @@
 #import <React/RCTUtils.h>
 #import <React/RCTView.h>
 #import <React/UIView+React.h>
-#import <reanimated/apple/LayoutReanimation/REAScreensHelper.h>
-#import <reanimated/apple/LayoutReanimation/REASnapshot.h>
+#import "REAScreensHelper.h"
+#import "REASnapshot.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
diff --git a/node_modules/react-native-reanimated/apple/reanimated/apple/REAModule.h b/node_modules/react-native-reanimated/apple/reanimated/apple/REAModule.h
index f4b45d4..969fe6b 100644
--- a/node_modules/react-native-reanimated/apple/reanimated/apple/REAModule.h
+++ b/node_modules/react-native-reanimated/apple/reanimated/apple/REAModule.h
@@ -5,7 +5,7 @@
 #import <React/RCTInitializing.h>
 #if REACT_NATIVE_MINOR_VERSION >= 74
 #import <React/RCTRuntimeExecutorModule.h>
-#import <ReactCommon/RCTRuntimeExecutor.h>
+#import <react/nativemodule/core/platform/ios/ReactCommon/RCTRuntimeExecutor.h>
 #endif // REACT_NATIVE_MINOR_VERSION >= 74
 #endif // REACT_NATIVE_MINOR_VERSION >= 77
 #import <rnreanimated/rnreanimated.h>
diff --git a/node_modules/react-native-reanimated/apple/reanimated/apple/REAUtils.m b/node_modules/react-native-reanimated/apple/reanimated/apple/REAUtils.m
index 78afbff..ad48a71 100644
--- a/node_modules/react-native-reanimated/apple/reanimated/apple/REAUtils.m
+++ b/node_modules/react-native-reanimated/apple/reanimated/apple/REAUtils.m
@@ -1,4 +1,4 @@
-#import <reanimated/apple/REAUtils.h>
+#import "REAUtils.h"
 
 @implementation REAUtils
 
